openadapt/entrypoint.py

```diff
@@ -0,0 +1,24 @@
+"""Entrypoint for OpenAdapt."""
+
+import multiprocessing
+
+if __name__ == "__main__":
```

Comment:
Does this need to happen before the imports below? If so perhaps worth mentioning in a comment, otherwise we can consolidate below.